category/categoryModel.js

```diff
@@ -0,0 +1,7 @@
+const mongoose = require('mongoose');
+
+const Category = mongoose.Schema({
+    title: {type: String, required: true}
+})
+
+module.exports = mongoose.model('Category', Category);
```

Comment:
might wanna change the Category Schema variable name to be something like categorySchema, 'Category' is a correct name for collection so keep that.